pkgs/development/python-modules/gplaycli/default.nix

```diff
@@ -30,17 +23,12 @@ buildPythonPackage rec {
     hash = "sha256-uZBrIxnDSaJDOPcD7J4SCPr9nvecDDR9h+WnIjIP7IE=";
   };
 
-  propagatedBuildInputs = [
```

Comment:
i'm not sure if these were meant to be or can be removed. when running `gplaycli --help`:

```
Traceback (most recent call last):
  File "/nix/store/8jhp71lj84skdv2kximr4fc52qqg1v4k-python3.11-gplaycli-3.29/bin/.gplaycli-wrapped", line 6, in <module>
    from gplaycli.gplaycli import main
  File "/nix/store/iqwkzq0i1wfrk2v8imd2vgx9v8qbm8af-python3.12-gplaycli-3.29/lib/python3.12/site-packages/gplaycli/gplaycli.py", line 29, in <module>
    from gpapi.googleplay import GooglePlayAPI, LoginError, RequestError
  File "/nix/store/gnz9m0p64mamphmjkhxnxl8jygwn897g-python3.12-matlink-gpapi-0.4.4.5/lib/python3.12/site-packages/gpapi/googleplay.py", line 6, in <module>
    from cryptography.hazmat.primitives.asymmetric.utils import encode_dss_signature
  File "/nix/store/p4m7znb6h8sbhbghf6scx8zv26c151n1-python3.12-cryptography-42.0.5/lib/python3.12/site-packages/cryptography/hazmat/primitives/asymmetric/utils.py", line 7, in <module>
    from cryptography.hazmat.bindings._rust import asn1
ImportError: cannot import name 'asn1' from 'cryptography.hazmat.bindings._rust' (unknown location)
```

looks like at least pyasn1 is still required here.